cli-commands/shape/options/framework-option.js

```diff
@@ -0,0 +1,21 @@
+const Option = require('../../option');
+const repositories = require('./../repositories.json');
+
+class FrameworkOption extends Option {
+    constructor() {
+        super(
+            'framework',
+            {
+                "describe": "The name of the framework that the project will be build up.",
```

Comment:
"The name of the framework that the project will be build up." => "A front-end framework a project will be prepared for you on"